package.json

```diff
@@ -29,7 +29,8 @@
     "react-router-dom": "^4.2.2",
     "redux": "^3.6.0",
     "redux-form": "^7.4.2",
-    "redux-promise-middleware": "^4.4.2",
+    "redux-merge-reducers": "0.0.2",
+    "redux-promise-middleware": "^4.2.0",
```

Comment:
@IAMOTZ why the version downgrade of the `redux-promise-middleware` ?

Reply:
That was not intentional, it probably happened when I was doing rebase to fix merge conflicts. I would update it back.
Thanks, @rhenshaw56.